docs/image/image.html

```diff
@@ -0,0 +1,169 @@
+                </li>
+                <li>
+                    <a class="sidebar-item" href="../card/card.html" rel="noopener noreferrer">Card</a>
+                </li>
+                <li>
+                    <a class="sidebar-item  link-active" href="../image/image.html" rel="noopener noreferrer">Image</a>
+                </li>
+                <li>
+                    <a class="sidebar-item" href="" rel="noopener noreferrer">Grid</a>
+                </li>
+                <li>
+                    <a class="sidebar-item" href="" rel="noopener noreferrer">Input</a>
+                </li>
+                <li>
+                    <a class="sidebar-item" href="" rel="noopener noreferrer">List</a>
+                </li>
+                <li>
+                    <a class="sidebar-item" href="" rel="noopener noreferrer">Modal</a>
+                </li>
+                <li>
+                    <a class="sidebar-item" href="" rel="noopener noreferrer">Navigation</a>
+                </li>
+                <li>
+                    <a class="sidebar-item" href="" rel="noopener noreferrer">Rating</a>
+                </li>
+                <li>
+                    <a class="sidebar-item" href="" rel="noopener noreferrer">Toast</a>
+                </li>
+                <li>
+                    <a class="sidebar-item" href="" rel="noopener noreferrer">Text</a>
+                </li>
+                
+            </ul>
+        </aside>
+        <main class="main-and-footer-container">
+
+            <div class="main-content">
+
+                <h1>Image</h1>
+                <p>Images are one of the significant elements of any application as they are used for a wide variety of purposes. They have a very important role in human-computer interaction.</p>
+
+                <h3>Responsive Image</h3>
+                <p>Images are extensively used in various applications and they are required in different sizes. The following images are responsive, that is, their size will change with respect to the size of the display.</p>
+
+                <div class="component-container">
+                    <img
+                        src="/assets/img1.jpg" alt="Responsive image"
+                        class="responsive-img"
+                    />
+                </div>
+
+                <div class="component-container">
+                    <img
+                        src="/assets/img2.jpg" alt="Responsive image"
+                        class="responsive-img"
+                    />
+                </div>
+
+                <p>The following code snippet contains the code for the above components.</p>
+
+                <div class="component-container code-snip-width">
+                    <iframe
+                        src="https://carbon.now.sh/embed/5wIEo6mGjLnZBbtvUcyF"
+                        style="width: 571px; height: 279px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
Kindly remove inline styles. You can add a class and the related styles on the class.